asm/lib_sram.s

```diff
@@ -0,0 +1,22 @@
+.section .text
+.syntax unified
+
+.include "include/gba.inc"
+
+@ SRAM Library
+@ Write 32-bit Integers to SRAM
+arm_func_start write_int_sram_fast
```

Comment:
like i said i think this should stay with the other asm functions for now

Reply:
Done.